### DISCUSSION TITLE
Can we install strapi using existing database

### DISCUSSION THREAD

**Discussion by izjenie** [strapi/strapi#5873] [Legacy Topics]
Helo,

Here's the case - I have strapi+mongodb already running.

Then I need to move the mongodb server, so I dump and restore to new mongodb in new server.

After that I was trying to install fresh new strapi to existing database with no luck.

Is there a way to move strapi installation from one place to another? Thanks.

**Comment by lauriejim** [comment]
Hello I'm not sure to 100% follow.

If you need to only migrate your MongoDB database without moving your Strapi app, you will just have to update the `config/environments/production/database.json` file to say to Strapi to connect the new database (which is a restore of the previous one)
